.github/workflows/send-system-notification.yml

```diff
@@ -0,0 +1,65 @@
+name: "Send system notification"
+
+on:
+  workflow_dispatch:
+    inputs:
+      channel:
+        description: "Name of channel to use. Must be defined in /infra/project-config"
```

Comment:
```suggestion
        description: "Name of channel to use. Must be defined in /infra/project-config/system-notifications.tf"
```

Reply:
that was intentional since I was just referring to the module not the specific file in case we rename the file, but you're right this is probably clearer will change